src/libraries/System.Security.Principal/README.md

```diff
@@ -0,0 +1,11 @@
+# System.Security.Principal
+
+This assembly no longer contains any code.  It is provided only to permit type unification for libraries built against previous versions of .NET.
+
+## Source
+
+* All types previously part of this library are now part of [System.Private.CoreLib](../System.Private.CoreLib/), exposed via [System.Runtime](../System.Runtime/).
```

Comment:
nit: should the link be more specific?

```suggestion
* All types previously part of this library are now part of [System.Private.CoreLib](../System.Private.CoreLib/src/System/Security/Principal), exposed via [System.Runtime](../System.Runtime/).
```

Reply:
Well, if the link was more specific the link text would need to be more specific.  While it probably works in this case, it just kinda feels weird to deeply link.